src/io/fabric8/Fabric8Commands.groovy

```diff
@@ -498,10 +498,10 @@ def getServiceURL(String serviceName, String namespace = null, String protocol =
 }
 
 def isOpenShiftS2I() {
-    def openshiftYaml = "target/classes/META-INF/fabric8/openshift.yml"
+  def openshiftYaml = findFiles(glob: '**/openshift.yml')
```

Comment:
I am not sure if we will have multiple files, as we can see above in the diff we were earlier just checking for one file in the same location now we are just checking for it recursively in the same directory.